src/test/java/org/jitsi/meet/test/mobile/base/MobileParticipantFactory.java

```diff
@@ -47,8 +47,8 @@ public MobileParticipantFactory(Properties config)
      * {@inheritDoc}
      */
     @Override
-    public Participant<? extends WebDriver> createParticipant(
-            String configPrefix, ParticipantOptions options)
+    public Participant<? extends WebDriver> doCreateParticipant(
```

Comment:
Does this need to stay public? The overriden method is protected

Reply:
right, that's by mistake